plugins/outputs/cloudwatch/cloudwatch.go

```diff
@@ -555,6 +555,7 @@ func BuildDimensions(mTags map[string]string) []*cloudwatch.Dimension {
 
 	for _, k := range keys {
 		if len(dimensions) >= MaxDimensions {
+			log.Printf("D! max MaxDimensions %v is less than than number of dimensions %v thus only taking the max number", MaxDimensions, len(dimensions))
```

Comment:
The wording in this log message seems off. If I have 100 dimensions, it would read:
```
D! max MaxDimensions 30 is greater than number of dimensions 100 thus only taking the max number
```

Reply:
Ah you changed it while I was reviewing. Fair enough